.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,32 @@
+name: Report a bug
+description: Something is not working as expected? Tell us the problem
+labels: ["kind/bug"]
+body:
```

Comment:
I am missing part:

```
title: 
```

Reply:
I think your thinking this as a placeholder, but it will be the value of the issue name (Title in the image)

I think is best to left it empty

![Screenshot from 2023-04-24 14-56-29](https://user-images.githubusercontent.com/66965232/234078084-999113b2-0e27-46bf-9fab-24fa801b7e81.png)